connector/src/main/java/org/geysermc/connector/utils/SkinProvider.java

```diff
@@ -209,7 +221,37 @@ private static Cape supplyCape(String capeUrl) {
         }
     }
 
-    private static BufferedImage scale (BufferedImage bufferedImage) {
+    private static BufferedImage readCapeFromJson(String url) throws IOException {
+        HttpURLConnection connection = null;
+        BufferedReader reader = null;
+        try {
+            connection = (HttpURLConnection) (new URL(url)).openConnection();
+            connection.setDoInput(true);
+            connection.setDoOutput(false);
+            connection.connect();
+            if (connection.getResponseCode() != 200) {
+                return null;
+            }
+            reader = new BufferedReader(new InputStreamReader(connection.getInputStream()));
```

Comment:
Could this not be changed to use `WebUtils.getBody` or add another function in there?